docs/get-started/README.md

```diff
@@ -9,7 +9,7 @@ that aim to make working with external overlays, such as
 
 # Installation
 
-`ReactMapGL` requires `node >= v4` and `react >= 15.4`.
+`ReactMapGL` requires `node >= v6.4.0` and `react >= 15.4`.
```

Comment:
What is the dependency on 6.4 again? 6 is a long time release, requiring 6.4 is pretty demanding. It is not only for building react-map-gl right, it is required just for installing it?